Automattic/newspack-plugin
# Update Grid component and wizards using it

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds column(s) and gutter support to the `Grid` and reorganises some of the Wizards currently using it (Dashboard, Site Design). Removes the use of `Grid` for the Components Demo.

Ultimately the goal is to use Grid throughout the Wizards, combined with Cards to split the content in multiple columns.

_Note: I have also remove the view buttons from the Dashboard -- we are going to stick with the grid for the moment._

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the different Wizards (especially Dashboard, Site Design, Components Demo)

### Other information:
